modules/dnn/src/layers/blank_layer.cpp

```diff
@@ -161,7 +161,11 @@ class BlankLayerImpl CV_FINAL : public BlankLayer
                                         const std::vector<Ptr<BackendNode> >& nodes) CV_OVERRIDE
     {
         auto ieInpNode = nodes[0].dynamicCast<InfEngineNgraphNode>()->node;
-        auto blank = std::make_shared<ov::op::v1::ConvertLike>(ieInpNode, ieInpNode);
+        // We use Reshape as blank layer.
+        // Previously, we used Concat layer, but it didn't work with scalar tensor.
+        // Also we used ConvertLike layer, but it did't work with old OpenVINO versions.
+        auto shape = std::make_shared<ov::op::v0::ShapeOf>(ieInpNode);
+        auto blank = std::make_shared<ov::op::v1::Reshape>(ieInpNode, shape, false);
         return Ptr<BackendNode>(new InfEngineNgraphNode(blank));
```

Comment:
May be just return `ieInpNode` without any "operations".

Reply:
Unfortunately, that won't work and workaround is to create a dummy node. I've checked that nGraph usually removes such kind of nodes as a part of middle graph processing (like identical Reshape, ConvertLike to the same type and so on).